### PR TITLE
Move cloud under pkg/cloud

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,6 +47,7 @@ depend-update:
 build: ## build binaries
 	$(DOCKER_CMD) go build $(GOGCFLAGS) -o bin/libvirt-actuator github.com/openshift/cluster-api-provider-libvirt/cmd/libvirt-actuator
 	$(DOCKER_CMD) go build $(GOGCFLAGS) -o bin/machine-controller github.com/openshift/cluster-api-provider-libvirt/cmd/machine-controller
+	$(DOCKER_CMD) go test $(GOGCFLAGS) -c -o bin/machines.test github.com/openshift/cluster-api-provider-libvirt/test/machines
```
Comment (paulfantom) [inline Makefile]: Why `test` in `build` stage?

Comment (ingvagabund) [reply Makefile]: it builds the test binary, it does not run the test

Comment (paulfantom) [reply Makefile]: /approve

**Pull request merged**